backend/src/graphql/resolver/AdminResolver.test.ts

```diff
@@ -1214,24 +1334,32 @@ describe('AdminResolver', () => {
                     email: 'peter@lustig.de',
                     amount: new Decimal(1900),
                     memo: 'Danke Peter!',
-                    creationDate: new Date().toString(),
+                    creationDate: creation
+                      ? creation.contributionDate.toString()
+                      : new Date().toString(),
                   },
                 }),
               ).resolves.toEqual(
                 expect.objectContaining({
                   errors: [
                     new GraphQLError(
-                      'The amount (1900 GDD) to be created exceeds the amount (500 GDD) still available for this month.',
+                      'The amount (1900 GDD) to be created exceeds the amount (1000 GDD) still available for this month.',
                     ),
                   ],
                 }),
               )
             })
+
+            it('logs the error thrown', () => {
+              expect(logger.error).toBeCalledWith(
+                'The amount (1900 GDD) to be created exceeds the amount (1000 GDD) still available for this month.',
+              )
+            })
           })
 
           describe('creation update is successful changing month', () => {
             // skipped as changing the month is currently disable
-            it.skip('returns update creation object', async () => {
+            it('returns update creation object', async () => {
```

Comment:
Please have this skipped. We do not want to have the month changed for now.

Reply:
fixed